### ISSUE TITLE
Can't update group settings

### ISSUE THREAD

**Issue by zefanja** [jay0lee/GAM#992]
Please confirm the following:
* I have upgraded to the latest GAM release from https://git.io/gamreleases and I still have this issue.
**Yes, latest release (4.90, Linux)**

* I am typing the command as described in the GAM Wiki at https://github.com/jay0lee/gam/wiki
**Yes.**

Full steps to reproduce the issue:
1. `gam update group myGroup who_can_post_message all_managers_can_post`

Expected outcome (what are you trying to do?):
**Set the right permission.**

Actual outcome (what errors or bad behavior do you see instead?):
**I get this:**

```
ERROR: value for whoCanPostMessage must be one of , 
-, NONE_CAN_POST:, The, group, is, disabled, and, archived., No, one, can, post, a, message, to, this, group., , 
-, When, archiveOnly, is, false,, updating, whoCanPostMessage, to, NONE_CAN_POST,, results, in, an, error., 
-, If, archiveOnly, is, reverted, from, true, to, false,, whoCanPostMessages, is, set, to, ALL_MANAGERS_CAN_POST., , 
-, ALL_MANAGERS_CAN_POST:, Managers,, including, group, owners,, can, post, messages., 
-, ALL_MEMBERS_CAN_POST:, Any, group, member, can, post, a, message., 
-, ALL_OWNERS_CAN_POST:, Only, group, owners, can, post, a, message., 
-, ALL_IN_DOMAIN_CAN_POST:, Anyone, in, the, account, can, post, a, message., , 
-, ANYONE_CAN_POST:, Any, Internet, user, who, outside, your, account, can, access, your, Google, Groups, service, and, post, a, message., Note:, When, whoCanPostMessage, is, set, to, ANYONE_CAN_POST,, we, recommend, the, messageModerationLevel, be, set, to, MODERATE_NON_MEMBERS, to, protect, the, group, from, possible, spam.. Got ALL_MANAGERS_CAN_POST.
```

**Comment by taers232c** [comment]
Basic Gam 4.90 can not update group settings due to Google change that broke the code.
I've submitted a fix to Jay to correct the problem but he hasn't applied it yet.
You can wait for Jay to apply the fix or you can use Advanced GAM (https://github.com/taers232c/GAMADV-XTD3)
that did not get broken by the change.

https://github.com/taers232c/GAMADV-XTD3/wiki/How-to-Upgrade-from-Standard-GAM

**Comment by zefanja** [comment]
fixed in 4.91 - thanks!
